server/app.js

```diff
@@ -0,0 +1,41 @@
+var createError = require('http-errors');
+var express = require('express');
```

Comment:
const instead of var

Reply:
This has been done in next pull request

Reply:
But we can't really merge this into master, this is not clean, either rebase from another pr or clean it :)

Reply:
done